src/components/project/NewProject/NewProject.tsx

```diff
@@ -45,10 +45,11 @@ const NewProject: FC<Props> = ({
   name,
 }) => {
   const [isActive, setIsActive] = useState(active);
-  const { createProject } = useProject();
+  const { createProject, projects } = useProject();
   const [isLoading, setIsLoading] = useState(false);
   const { createLog } = useLogActivity();
   const { open: openTab } = useFileTab();
+  const [currentProjectType, setCurrentProjectType] = useState(projectType);
```

Comment:
`currentProjectType` seems to be a misnomer: it's not a type of current project, but type of new project to be created.